### ISSUE TITLE
Incorrect display of Cyrillic characters in hints from XML documentation

### ISSUE THREAD

**Issue by leon-ts** [X-Sharp/XSharpPublic#504]
There is a problem with displaying Cyrillic characters (Ukraine, Russian, Belarusian) from XML documentation in hints in the code.

Screenshot 1 shows how the method source code (method declaration) with XML documentation looks and appears (correct).
![1 - View in Source Code](https://user-images.githubusercontent.com/64518775/95740957-16ad0e80-0c96-11eb-9ae4-26b44fdd417d.png)

Screenshot 2 shows what the tooltip looks like in other code that uses the above method (not correct).
![2 - View in External Code (Same Assembly)](https://user-images.githubusercontent.com/64518775/95740964-1a409580-0c96-11eb-884a-1037b8210a83.png)

All code is located in one assembly.

An additional complication of this problem is that it does not always occur. Once I load the project - the problem described above is there. Another time I load the project - there is no problem, all the tips are displayed correctly. I've wanted to send you this issue for a long time, but I couldn't always reproduce it. Now it has appeared and I made screenshots.

**Comment by leon-ts** [comment]
IDE: VS2019

**Comment by leon-ts** [comment]
The problem manifests itself in all versions of XSharp that I have used since spring 2020.

**Comment by RobertvanderHulst** [comment]
Leonid,
Can you create a small example, so I can see if this is caused by source in UTF8 format, or by source in Ansi format with a cyrillic codepage.

Robert

**Comment by leon-ts** [comment]
Robert,
I created a small project, but I cannot reproduce the problem.
As I mentioned in the description of the problem, it occurs one time out of ten (conditionally). Even on the project where I took screenshots, I closed the project and reopened and the problem does not reproduce. I even rebooted my computer, opened the project and the problem is not reproducible.

Nevertheless, I attached a small project, which has XML-documentation in Cyrillic.
[ClassLibrary1.zip](https://github.com/X-Sharp/XSharpPublic/files/5365070/ClassLibrary1.zip)
